tests/test_main.py

```diff
@@ -34,3 +34,8 @@ def test_video_unavailable(get):
     )
     with pytest.raises(VideoUnavailable):
         youtube.prefetch()
+
+
+def test_video_title_extraction():
```

Comment:
One problem with this test that I ran into myself previously -- you can't actually create YouTube objects in Travis CI and get consistent results without using `mock` to mock `pytube.request.urlopen`. YouTube semi-regularly returns 429 responses made from Travis CI, so we need to mock the http responses so no connections actually get made to the youtube servers. I'm happy to rewrite this to use mock requests in a future PR.

Reply:
Actually, now that I look at it, there's an existing title test in the code that can be repurposed to accomplish this better. It previously tested two possible execution paths, one of which no longer exists. The current test isn't great though, so I can rework that